cmd/utils/dev.go

```diff
@@ -197,7 +197,7 @@ func SelectDevFromManifest(manifest *model.Manifest, selector OktetoSelectorInte
 		}
 		return devs[i] < devs[j]
 	})
-	items := []SelectorItem{}
+	var items []SelectorItem
```

Comment:
✨ fix suggested by the linter [[1]](https://deepsource.io/gh/okteto/okteto/run/851a6b0a-999a-4ffb-89ed-cd9c008f703b/go/GO-W1027?listindex=0)